Exercise8_Kroeze.R

```diff
@@ -0,0 +1,41 @@
+## Nathan Kroeze
+## Exercise 8
+## 10 November 2022
+
+setwd("C:/Users/Natha/Desktop/Biocomputing/Exercise08/")
+
+library(dplyr)
+
+iris <- read.csv("iris.csv")
+
+
+## Q1 Replicate the head function in R by defining a function
+ ## Requires inputs inlcuding the data to be subset and the number of of rows to subset
+head_exercise <- function(data,n){
+  out <- data[1:n,]
+  return(out)
+}
+
+## Q2 R Tasks
+## a) Print the last two rows in the last two columns
+ ## Two solutions, one "hardcoded" and one that introduces flexibility
+iris[149:150,4:5]
+iris[(nrow(iris)-1):nrow(iris),(ncol(iris)-1):ncol(iris)]
+
+## b) Number of observations for each species in the dataset
+iris %>% count(Species)
+
+## c) Get rows with Sepal.Width > 3.5
+wide_iris <- subset(iris,Sepal.Width > 3.5)
+
+
+## d) Write the data for species setosa to a csv file
+setosa_iris <- subset(iris,Species == "setosa")

```

Comment:
we used [] to extract and manipulate the data for now. dplyr package and subset function are not needed

Reply:
-0.25

Reply:
I have a further discussion with Dr Jones and he said this will be Ok
+0.25